### PR TITLE
iscroll-x attribute to add any iscroll option

### PR DESCRIPTION
**Pull request by IgorRubinovich**
a couple of changes:
- added any attribute of the form `iscroll-x` to refer to parameter `x` of iscroll (e. g. `iscroll-snap="div"`). 
- `.next` and `.prev` map to iscroll's
- fixed default `scroll-x` to be set up in `ready`, since when a boolean property is set to `true` by default it will always be true regardless of user settings

Hope you will take these changes, and thanks for starting this.

### CODE REVIEW

File: rv-iscroll.html [hunk 1]
```diff
@@ -125,23 +134,56 @@
...
+      scrollY: {
+        type: Boolean,
+        // value: true // ir: depreciating - boolean values are true whenever set so it's not possible to override with something like scroll-y="false"
       },
 
     },
 
   ready: function () {
+	if(!this.scrollX && !this.scrollY)
+		this.scrollX = true;
```
Comment (yogaraptor) [inline rv-iscroll.html]: [iScroll docs](https://github.com/cubiq/iscroll#optionsscrollxoptionsscrolly) imply this should be `this.scrollY =  true`

**Comment by yogaraptor** [comment]
Thanks very much for this Igor (I just got back from a break so sorry for the delay). Will happily merge this pending a couple of niggles:
1) Good catch on the default `true`s, but [the iScroll docs](https://github.com/cubiq/iscroll#optionsscrollxoptionsscrolly) specify that `scrollX` is `false` by default and `scrollY` is `true`. If you could adjust your code to match this that would be great (have annotated the commit for you)
2) Convert indents to two spaces rather than tabs in keeping with the rest of the code (I usually use hard tabs too but the polymer element seed uses two spaces - I may convert this en-masse later)
3) Remove any console.log calls

**Comment by IgorRubinovich** [comment]
no problem will make the changes

in the meanwhile though I figured .next and .prev are not nessessary, it's probably easier to (1) rename myScroll to something like say .scroll, and then (2) just use it as in el.scroll.next(), this way the element becomes really just a thin wrapper and scope keeper rather than anything functional 

what do you reckon?

**Comment by yogaraptor** [comment]
Yeah, I think that sounds good - makes upgrading iScroll painless. If you want to go ahead and sort that I'll merge this in.

**Pull request closed without merging**